### ISSUE TITLE
ArchUnit: Allow to customize default architecture styles

### ISSUE THREAD

**Issue by stefanzilske** [xmolecules/jmolecules-integrations#291]
Hey, I am using jMolecules and ArchUnit to verify my project's structure. I prefer to use jMolecules annotations in `package-info.java` files.

However, I would like customize the default layouts, e.g. LayeredStrict. I would like to allow API to access DOMAIN directly, but I don't want to use the simple Layered layout, which allows a lot more dependencies.

I cannot really customize that, but still use the jMolecules annotation because many things `JMoleculesArchitectureRules` are private or package protected and cannot really work around them.

So, tl;dr - my question is: Can I customize the allowed dependencies, but still use jMolecules annotations? I don't wan't to switch to defining packages in the ArchUnit test, that define the layers.

Cheers!
Stefan
